xmlserializer/CMakeLists.txt

```diff
@@ -53,11 +53,16 @@ if(NOT XML2_XINCLUDE_SUPPORT)
             "please install a version of libxml2 supporting it.")
 endif()
 
-include_directories(
-    ${LIBXML2_INCLUDE_DIR}
-    "${PROJECT_SOURCE_DIR}/utility")
+target_include_directories(xmlserializer
+    PRIVATE ${LIBXML2_INCLUDE_DIR}
```

Comment:
Check if ${LIBXML2_INCLUDE_DIR} is not part of libxml2's INTERFACE_INCLUDE_DIRECTORIES.
If it is, remove this line.